src/Paramore.Brighter.MessagingGateway.AWSSQS/SqsMessagePublisher.cs

```diff
@@ -73,5 +72,16 @@ public string Publish(Message message)
 
              return null;
         }
+
+        private static string GetSubject(Message message)
+        {
+            var subjectExists = message.Header.Bag.TryGetValue("Subject", out var subject);
+            if (subjectExists)
+            {
+                message.Header.Bag.Remove("Subject");
```

Comment:
I considered not removing this from the bag meaning it would be added to message attributes but that would make the test asserting the subject is produced and consumed correctly meaningless